plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py

```diff
@@ -55,7 +55,8 @@
     aws_not_configured = True
 
 
-ami = os.environ.get("AWS_RAY_AMI", "ami-099b85625184ce698")
+# ami = os.environ.get("AWS_RAY_AMI", "ami-0c92175410c7505e3")
+ami = "ami-0c92175410c7505e3"
```

Comment:
intentional?

Reply:
Yes, for testing purpose. I do not want to update CIrcleCI env variable before I can verifiy the new AMI works :)

will revert this back after updating the AMI on CircleCI side.